source/tables.tex

```diff
@@ -487,21 +469,17 @@
  \newcommand{\hdstyle}[1]{\textbf{##1}}
  \newcommand{\rowhdr}[1]{\hdstyle{##1}&}
  \newcommand{\colhdr}[1]{\multicolumn{1}{|>{\centering}m{#6}|}{\hdstyle{##1}}}
- %% FIXME: figure out a way to reuse floattable here
```

Comment:
Awesome! I've been secretly waiting for this FIXME to be resolved for 3 years. :) (https://github.com/Eelis/cxxdraft-htmlgen/issues/5)